src/core.js

```diff
@@ -850,8 +854,24 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     }
   }
 
+  /**
+   * Execute batch of operations with updating cache only when necessary. Function is responsible for renewing row index mapper's and column index mapper's
```

Comment:
Here I have a suggestion to keep comment line no wider than 120 chars. Now the longest comment line is 167 columns wide, which makes me scroll horizontally.

Reply:
I changed the longest ones: https://github.com/handsontable/handsontable/pull/5945/commits/c0b277af15e570588818c7ba4e6dcc980f38f8f9